src/components/Modal/staking-hub/AddAddressToERC1820RegistryModal.tsx

```
@@ -106,14 +128,34 @@ const AddAddressToERC1820RegistryModal = ({
+  const createAndExecuteTx = async () => {
+    set_error(null);
+    set_txStarted(true);
+    if (signer && safeAddress) {
+      return dispatch(
+        safeActionsAsync.createAndExecuteSafeContractTransactionThunk({
+          data: TXdata,
+          signer,
+          safeAddress,
+          smartContractAddress: ERC1820_REGISTRY,
+        })
+      )
```

ESLint

[error] Missing trailing comma.

(comma-dangle)